contrib/friendz/src/friendz_android.nit

```diff
@@ -35,3 +45,12 @@ redef class AndroidPointerEvent
 	redef fun x do return super / app.zoom
 	redef fun y do return super / app.zoom
 end
+
+redef class Game
+	redef fun load_sounds(app: App)
+	do
+		super
+                app.default_mediaplayer.looping = true
+		if not self.music_muted then music.play
```

Comment:
I do not understand why I have something specific to do on android that cannot be done in app.nit